jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/ContextHandler.java

```diff
@@ -631,6 +637,57 @@ protected void doStart() throws Exception
         contextInitialized();
     }
 
+    @Override
```

Comment:
FYI I think you've also fixed this and lines 277 297 in your other PR about changes to Handler.